examples/crna-kitchen-sink/package.json

```diff
@@ -30,8 +30,8 @@
     "preset": "jest-expo"
   },
   "dependencies": {
-    "expo": "^19.0.0",
+    "expo": "19.0.0",
     "react": "16.0.0-alpha.12",
-    "react-native": "^0.46.1"
+    "react-native": "0.46.1"
```

Comment:
I really don't see the benefit of locking our example apps. If a new minor version is released that breaks something, we really have no way of knowing. I know a few of us run these example apps to test if anything breaks before merging or publishing.

Reply:
I understand your reasoning @Hypnosphi.

Generally I'd rather have errors all over the place when we're not compatible with latest.
This will keep us alert and nudges us to resolve it quickly.

It's annoying for PR's but PR branches can always lock it down if the problem comes up.

Having said that, I think in the case of ReactNative we want to be extra cautious. Reactnative itself is on the 0.x.y versioning and is actually using pre-released version of other packages. Locking them down sounds sane.

I'm really interested in what @shilman @tmeasday think about this.

Reply:
Definitely on the side of keeping up with latest. It will cause some headaches occasionally, but overall I think it will help us much more than it will hurt us.

Reply:
@ndelangen @shilman isn’t it a task for greenkeeper or similar tool?